packages/protocol/src/helpers/SystemAccessControl.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.15;
+
+/**
+ * @title SystemAccessControl.
+ * @author Fujidao Labs
+ * @notice Helper contract that should be inherited by contract implementations that
+ * should call {Chief} for access control checks.
+ */
+
+import {IChiefAccessHelper} from "../interfaces/IChiefAccessHelper.sol";
+import {CoreRoles} from "./CoreRoles.sol";
+
+contract SystemAccessControl is CoreRoles {
```

Comment:
Should it be an abstract contract? 
I also suggest placing this contract and all the others in a new directory "access".

Reply:
Abstract contracts are only ones that do not implement certain functions. In this case SystemAccessControl is a very simple contract (or class) that just inherits the core roles and makes available generic modifiers that will be used across different contracts. It is not required to be marked as 'abstract'.

I created the folder "access" 
access/
├── CoreRoles.sol
├── SystemAccessControl.sol
└── TimeLock.sol